programs/bpf_loader/src/syscalls.rs

```diff
@@ -59,14 +60,18 @@ pub enum SyscallError {
     MalformedSignerSeed(Utf8Error, Vec<u8>),
     #[error("Could not create program address with signer seeds: {0}")]
     BadSeeds(PubkeyError),
-    #[error("Program id is not supported by cross-program invocations")]
-    ProgramNotSupported,
+    #[error("Program {0} supported by inner instructions")]
```

Comment:
Wanted to be specific so folks don't get the wrong impression about why the program is not supported.  Gonna leave it because it is only used for inner instructions

Reply:
You changed it to say that it _is_ supported, should be fixed to at least say `unsupported` right?